src/main/java/life/qbic/userdb/model/ProjectInfo.java

```diff
@@ -0,0 +1,102 @@
+package life.qbic.userdb.model;
+
+public class ProjectInfo {
+
+  private String description;
+  private String secondaryName;
+  private Person investigator;
+  private Person contact;
+  private Person manager;
+  private boolean isPilot;
+  private String space;
+  private String projectCode;
+  private int projectID;
+
+  public ProjectInfo(String space, String code, String description, String secondaryName,
+      boolean isPilot, Person investigator, Person contact, Person manager) {
+    this.space = space;
+    this.projectCode = code;
+    this.description = description;
+    this.secondaryName = secondaryName;
+    this.isPilot = isPilot;
+    this.investigator = investigator;
+    this.contact = contact;
+    this.manager = manager;
+  }
+
+  public ProjectInfo(String space, String code, String description, String secondaryName, int id) {
+    this.space = space;
+    this.projectCode = code;
+    this.description = description;
+    this.secondaryName = secondaryName;
+    this.projectID = id;
+    this.isPilot = false;
+  }
```

Comment:
Same as in Person, I'd recommend using a builder pattern to clearly seperate needed and additional fields here